packages/squiggle-lang/src/rescript/Reducer/Reducer_Dispatch/Reducer_Dispatch_BuiltIn.res

```diff
@@ -43,16 +43,35 @@ let callInternal = (call: functionCall): result<'b, errorValue> => {
     | None => RERecordPropertyNotFound("Record property not found", sIndex)->Error
     }
 
+  let inspect = (value: expressionValue) => {
+    Js.log(value->toString)
+    value->Ok
+  }
+
+  let inspectLabel = (value: expressionValue, label: string) => {
+    Js.log(`${label}: ${value->toString}`)
+    value->Ok
+  }
+
+  let inspectPerformance = (value: expressionValue, label: string) => {
+    let _ = %raw("{performance} = require('perf_hooks')")
+    let start = %raw(`performance.now()`)
+    let finish = %raw(`performance.now()`)
+    let performance = finish - start
+    Js.log(`${label}: ${value->toString} performance: ${Js.String.make(performance)}ms`)
+    value->Ok
```

Comment:
This is pretty neat, thanks!

Reply:
Just a reminder. It is not able to measure fine grained time like a few milliseconds (No way, intentionally prevented in Javascript). But fine for huge computations.